app.py

```
@@ -27,14 +36,17 @@ def add_header(response):
 
 @app.route('/')
 def index():
+    # Render homepage
     return render_template('index.html')
 
 @app.route('/progress')
 def get_progress():
```

Ruff (0.8.0)

Missing return type annotation for public function `get_progress`

(ANN201)